anyway/parsers/cbs/executor.py

```diff
@@ -1043,33 +1047,34 @@ def receive_rollback(conn, **kwargs):
 
 
 def create_tables():
+    chunk_size = 50000
     try:
         with db.get_engine().begin() as conn:
             event.listen(conn, "rollback", receive_rollback)
-            logging.debug("begin create tables")
-            conn.execute("TRUNCATE involved_markers_hebrew")
-            logging.debug("after TRUNCATE involved_markers_hebrew")
-            conn.execute("TRUNCATE vehicles_markers_hebrew")
-            logging.debug("after TRUNCATE vehicles_markers_hebrew")
-            conn.execute("TRUNCATE vehicles_hebrew")
-            logging.debug("after TRUNCATE vehicles_hebrew")
-            conn.execute("TRUNCATE involved_hebrew")
-            logging.debug("after TRUNCATE involved_hebrew")
-            conn.execute("TRUNCATE markers_hebrew")
-            logging.debug("after TRUNCATE markers_hebrew")
-            conn.execute("INSERT INTO markers_hebrew " + VIEWS.MARKERS_HEBREW_VIEW)
-            logging.debug("after INSERT INTO markers_hebrew")
-            conn.execute("INSERT INTO involved_hebrew " + VIEWS.INVOLVED_HEBREW_VIEW)
-            logging.debug("after INSERT INTO involved_hebrew")
-            conn.execute("INSERT INTO vehicles_hebrew " + VIEWS.VEHICLES_HEBREW_VIEW)
-            logging.debug("after INSERT INTO vehicles_hebrew ")
-            conn.execute(
-                "INSERT INTO vehicles_markers_hebrew " + VIEWS.VEHICLES_MARKERS_HEBREW_VIEW
-            )
-            logging.debug("after INSERT INTO vehicles_markers_hebrew")
-            conn.execute(
-                "INSERT INTO involved_markers_hebrew " + VIEWS.INVOLVED_HEBREW_MARKERS_HEBREW_VIEW
-            )
+            truncate_table_for_connection(conn, AccidentMarkerView)
+            run_query_and_insert_to_table_in_chunks(VIEWS.create_markers_hebrew_view(), AccidentMarkerView,
+                                                    AccidentMarker.id, chunk_size, conn)
```

Comment:
Hi Tamar, are these two actions occur in same transaction? If not, how long you think the system will be with empty table?

Reply:
they do happen in the same transaction, as it happens today. actually, I believe that we have this situation of reading empty tables mid-transaction because we use "truncate" instead of "delete". [more info](https://www.postgresql.org/docs/current/mvcc-caveats.html)
so, I believe using Delete will make it unnecessary to use the temporary tables we discussed.

Reply:
I already updated the PR to use Delete.